rma_sale/static/src/js/rma_portal_form.esm.js

```diff
@@ -0,0 +1,125 @@
+/** @odoo-module **/
+/* Copyright 2021 Tecnativa - David Vidal
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.PortalRmaSale = publicWidget.Widget.extend({
+    selector: "#form-request-rma",
+    events: {
+        "change .rma-operation": "_onChangeOperationId",
+        "change #delivery-rma-qty input": "_onChangeQty",
+        "click .o_rma_portal_shipping_card": "_onChangeShippingAddress",
+    },
+
+    /**
+     * @override
+     */
+    start: function () {
+        const ids = this.$("[name*='-operation_id']")
+            .map(function () {
+                return this.name.replace("-operation_id", "");
+            })
+            .get();
+        this.$submit = $("#form-request-rma button[type='submit']");
+        this.rows_ids = ids;
+        // We'll build an object that will ease the form check. It could be further
+        // extended with additional checks.
+        this.rows = {};
+        $.each(ids, (id) => {
```

Comment:
Whatch out :eyes:  `$.each` isn't a drop in replacement for `_.each` and here it's behaving completly different and it was inducing errors (and as git wasn't catching the diff for this file it was tricky to spot why).

Here's the [right replacement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach), as we want to iterate over the array elements (it's recommend to try to replace the underscore stuff with vanilla js when possible)
```suggestion
        ids.forEach((id) => {
```